docs/pages/eas-update/build-locally.mdx

```diff
@@ -68,7 +68,15 @@ Inside the app config, make changes to use [`runtimeVersion`](/versions/latest/c
   }
 ```
 
-> In the [introduction guide](/eas-update/getting-started), the channel name is set by EAS Build and is different for various build profiles. Since this is a local build, it is necessary to set this request header explicitly in the app config.
+> In the [introduction guide](/eas-update/getting-started), the channel name is set by EAS Build and is different for various build profiles and automatically synchronized on the server. Since this is a local build, it is necessary to set this request header explicitly in the app config and also necessary to create the channel on the server manually.
```

Comment:
```suggestion
> In the [Get started](/eas-update/getting-started) guide, the channel name is set by EAS Build and is different for various build profiles and automatically synchronized on the server. Since this is a local build, it is necessary to set the request header explicitly in the app config, as shown above, and then manually create the channel on the server.
```